query-algebrizer/src/cc.rs

```diff
@@ -69,7 +69,7 @@ pub enum DatomsColumn {
 
 impl DatomsColumn {
     pub fn as_str(&self) -> &'static str {
-        use DatomsColumn::*;
+        use DatomsColumn::{Attribute, Entity, Tx, Value, ValueTypeTag};
```

Comment:
I agree. I think it's fine to import * in *small* contained scopes like function bodies.